tony-core/src/main/java/com/linkedin/tony/TonyApplicationMaster.java

```diff
@@ -98,6 +98,7 @@
   private String appIdString;
   private String amHostPort;
   private FileSystem fs;
+  private FileSystem historyFs;  // FileSystem used to write history-related files like config and events
```

Comment:
is this used for writing to another namenode?

Reply:
Yes. I'll update the comment to make it more clear.